File: lista_de_exercicios/questao6/questao6.go
package main

import "fmt"

type Motor struct {
    Potencia int
}

type Carro struct {
    Modelo string
    Motor  Motor
}

func main() {
    motor := Motor{Potencia: 200}
    carro := Carro{Modelo: "Sedan", Motor: motor}
    fmt.Println(carro.Modelo, carro.Motor.Potencia)
}

File: lista_de_exercicios/questao6/questao6.py
class Motor:
    def __init__(self, potencia):
        self.potencia = potencia

class Carro:
    def __init__(self, modelo, motor):
        self.modelo = modelo
        self.motor = motor

motor = Motor(200)
carro = Carro("Sedan", motor)
print(carro.modelo, carro.motor.potencia)
